app-backend/batch/src/main/scala/stacExport/WriteStacCatalog.scala

```diff
@@ -195,7 +195,7 @@ final case class WriteStacCatalog(exportId: UUID)(
 
     dbIO.transact(xa) flatMap {
       case (exportDef, Some(layerInfoMap)) =>
-        val currentPath = s"s3://$dataBucket/stac-exports"
+        val currentPath = s"s3://$dataBucket"
```

Comment:
This isn't exposed to users and it keeps our bucket a little more sane by at least putting all of the exports in a "folder" on S3

Reply:
I think it _is_ exposed to users -- with this included, when you unzip it, everything is in a `stac-exports` folder

Reply:
Also, the s3 path that results is still `s3://rasterfoundry-development-data-us-east-1/stac-exports/<id>/catalog.zip` -- one thing that made this a really hard +16/-13 PR is that there are tons of things that look feel like they should know how to depend on the data that have no clue. This is one of them, the relative path from the items to the catalog root or the collections to the catalog root being fixed is another. A lot of data here are just magic strings that exist to serve sort of obscure purposes.

Reply:
geez but the value in the database doesn't have it :fearful: digging a little more

Reply:
Ah woops ok so:

It _doesn't_ go under `stac-exports` on s3, that was just in my shell history. However, it also is exposed to users -- it definitely gets included in the zip. I'm fine saying humans are just going to have to deal or that we're _actually helping_ by making it so that they can unzip several of our exports from the same location and have everything nicely organized afterward.

Reply:
Re-exported and confirmed everything still ties together, merging